pkgs/development/python-modules/pytest-testinfra/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, ansible-core
+, paramiko
+, pytest
+, pytestCheckHook
+, pytest-xdist
+, pywinrm
+, salt
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-testinfra";
+  version = "8.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-m0CCi1j7esK/8pzBRlk0rfQ08Q3VoQj2BTXe5SZgpj0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools-scm
+  ];
+
+  nativeCheckInputs = [
+    ansible-core
+    paramiko
+    pytest
+    pytestCheckHook
+    pytest-xdist
+    pywinrm
+    salt
+  ];
+
+  preCheck = ''
+    export HOME=$(pwd)
+  '';
+
+  pytestFlagsArray = [
+    "--disable-pytest-warnings"
+  ];
+
+  disabledTests = [
+    # test/test_backends.py
+    "test_command"
+    "test_encoding"
+    "test_ansible_any_error_fatal"
+    "test_user_connection"
+    "test_sudo"
+    "test_docker_encoding"
+  ];
+
+  disabledTestPaths = [
+    "test/test_modules.py"
+  ];
```

Comment:
We should comment why those tests do not work. For example the sudo test likely does not work because there is no sudo in the sandbox and cannot be.